app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Wrapper for launching etcd via rkt.
+#
+# The version of etcd to be used can be set through ETCD_VERSION. The version
+# must be set to one of the tags published here:
+#
+#   https://github.com/coreos/etcd/releases
+#
+# If no version is provided, a default version of etcd is used.
+
+set -e
+
+ETCD_VERSION="${ETCD_VERSION:-v2.3.6}"
+ETCD_ACI="${ETCD_ACI:-coreos.com/etcd}"
```

Comment:
`ETCD_ACI="${ETCD_ACI:-coreos.com/etcd:$ETCD_VERSION}"`, then later on only use `ETCD_ACI`. This allows to supply a local file which could also be prefetched and/or baked into the CoreOS image.